### PR TITLE
Ab order

### PR DESCRIPTION
**Pull request by djbrownbear**
add questions 5 - 11 to order form

### CODE REVIEW

File: orders.html [hunk 1]
```diff
@@ -90,14 +90,69 @@ <h1>Orders</h1>
...
+												<option value="2">Chocolate</option>
+											</select>
+										</label>
+
+										<label class="dietRestrictions">
+											Please list any allergies or dietary restrictions
+											<input type="text" name="dietRestrictions" placeholder="eggs, soy, wheat, etc. (leave blank if n/a)">
+										</label>
+
+										<label class="colorPalette">
```
Comment (djbrownbear) [inline orders.html:124]: this is my first take on the color palette. it is optional but i wonder if the average user could handle this.

other thoughts:  link to their party theme/inspiration

**Comment by djbrownbear** [comment]
so far, I focused on functionality of the form and not aesthetics...

(i.e. if we go the current color palette route, I envision that the three colors would just be listed in one row to pick)

File: orders.html (hunk 1 above)
Comment (Yu-HuanWu) [reply orders.html:124]: i like the color palette but i wonder if that will overwhelm average users. Let's wait for Amanda to respond to my email and we can ask her about this feature.

File: assets/js/util.js [hunk 2]
```diff
@@ -17,6 +17,36 @@
...
+			$(".address").css("display", "none");
+		}
+		return false;
+	});
+
+	/**
+   *  Limit the order form date picker to today's date and beyond
+   *  source: https://answers.laserfiche.com/questions/165725/Only-allow-future-dates-in-date-field
+   */
+  $(document).ready(function () {
```
Comment (Yu-HuanWu) [inline assets/js/util.js:38]: this is really cool, i didn't realize that html can't detect today's date on its own without js.

**Review by Yu-HuanWu — APPROVED** [review]
lgtm

**Pull request merged**